core/tests/puppeteer-acceptance-tests/specs/moderator/edit-featured-activities-list.spec.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance tests for the functionality of editing the
+ * featured activities list by a moderator.
+ */
+
+import {UserFactory} from '../../utilities/common/user-factory';
+import testConstants from '../../utilities/common/test-constants';
+import {Moderator} from '../../utilities/user/moderator';
+import {ExplorationEditor} from '../../utilities/user/exploration-editor';
+import {LoggedOutUser} from '../../utilities/user/logged-out-user';
+
+const DEFAULT_SPEC_TIMEOUT_MSECS: number =
```

Comment:
Why did you specify "number" here? Is it necessary? I don't see this definition in other spec files.

Reply:
sure, removed for consistency.